packages/grafana-ui/src/components/Button/styles.ts

```diff
@@ -111,7 +106,6 @@ export const getButtonStyles = stylesFactory(({ theme, size, variant, withIcon }
       font-family: ${theme.typography.fontFamily.sansSerif};
       line-height: ${theme.typography.lineHeight.xs};
       padding: ${padding};
-      text-align: ${withIcon ? 'left' : 'center'};
```

Comment:
This wasn't actually needed cause there was no more space in the button that the contents needed